Gemfile.lock

```diff
@@ -78,29 +78,29 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
-    annotate (3.0.3)
+    annotate (3.1.1)
```

Comment:
### [annotate](http://github.com/ctran/annotate_models)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.0.3 → 3.1.1

[_~~change-log~~, [source-code](https://github.com/ctran/annotate_models)_]





<details>
<summary>Commits</summary>

A change of **76** commits. See the full changes on [the compare page](https://github.com/ctran/annotate_models/compare/v3.0.3...v3.1.1).

These are the first 10 commits:
- (4f7c7d5) [Merge branch 'release/3.0.3' into develop](https://github.com/ctran/annotate_models/commit/4f7c7d5d0c022bb141d6935ba6d8422f2361802a)
- (5da92c4) [Fix Github release action (#682)](https://github.com/ctran/annotate_models/commit/5da92c47a0c7e03a8b10339be7b3a29797d0eda9)
- (693abdd) [Fix shifted when format_markdown option enabled and used non…](https://github.com/ctran/annotate_models/commit/693abdd298c2bdce5fb931d9c7dbb1d10b560fa1)
- (a816255) [FIX: Ensure only one line is around the annotation (#669)](https://github.com/ctran/annotate_models/commit/a8162558cf8d1bec86df6b7ef8a438b9bc41f488)
- (1df2153) [[Fix #675] Correct indentation for double-byte characters (#…](https://github.com/ctran/annotate_models/commit/1df2153fd70a3ee2722519401b08d991cdf8ba8c)
- (a4be28a) [[Fix #624] Correct default values for columns when ActiveRec…](https://github.com/ctran/annotate_models/commit/a4be28adb923817792ce937a3c1ef6615dcf1a06)
- (23c56ec) [Fix Serializers Test Directory (#625)](https://github.com/ctran/annotate_models/commit/23c56ec725f9745548d45c390bb318de259f5916)
- (5da7964) [Add upgrade instructions to README (#687)](https://github.com/ctran/annotate_models/commit/5da79649c6214e04d4d536465966e811a642ee8e)
- (d84c526) [Fix foreign key issue with Rails 6 and Sqlite3 (#695)](https://github.com/ctran/annotate_models/commit/d84c52656f8f9b0945eb4aff41d523b5fb17590d)
- (2f0cb8c) [Add ActiveAdmin option to template (#693)](https://github.com/ctran/annotate_models/commit/2f0cb8cbd13bf36f140c3a6eaddc00a1f5b42ae1)


</details>